repos/system_upgrade/el7toel8/models/grubenv.py

```diff
@@ -0,0 +1,13 @@
+from leapp.models import Model
+from leapp.topics import SystemFactsTopic
+
+
+class HybridImageDetected(Model):
+    """
+    Model used for instructing Leapp to convert "grubenv" symlink
+    into a regular file in case of hybrid (BIOS/EFI) images using BIOS
+    on Azure.
+
+    This model will likely be removed in the future.
+    """
+    topic = SystemFactsTopic
```

Comment:
Really do not produce *empty* msg. Instead:
```suggestion
    topic = SystemFactsTopic
    
    detected = fields.Boolean(default=False)
```

In such case, maybe even the name of the class could be changed, like `HybridImageDetected`. The point is, that when you detect it, you should create non-default msg. Maybe it's *inherited* from the `TransactionCompleted` model, but that one has different purpose than this new one.